src/main/java/frc/robot/commands/PathWeaverCommand.java

```diff
@@ -0,0 +1,60 @@
+package frc.robot.commands;
+
+import java.io.IOException;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryUtil;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.FalconDriveSubsystem;
+
+public class PathWeaverCommand extends CommandBase {
+  private static final String TRAJECTORY_PATH = "paths/test.path";
+  private Trajectory trajectory;
+  private double startTime = 0;
+  private final FalconDriveSubsystem subsystem;
+  private final RamseteController controller = new RamseteController();
+
+  public PathWeaverCommand(FalconDriveSubsystem subsystem) {
```

Comment:
Might be a good idea to take the Path as a constructor argument. Makes the code usable with multiple different paths, including potentially paths calculated on-the-fly